src/GitVersionExe/ArgumentParser.cs

```diff
@@ -263,25 +263,43 @@ public static Arguments ParseArguments(List<string> commandLineArguments)
 
                     if (arguments.UpdateAssemblyInfoFileName.Count > 1 && arguments.EnsureAssemblyInfo)
                     {
-                        throw new WarningException("Can't specify multiple assembly info files when using -ensureassemblyinfo switch, either use a single assembly info file or do not specify -ensureassemblyinfo and create assembly info files manually");
+                        throw new WarningException("Can't specify multiple assembly info files when using /ensureassemblyinfo switch, either use a single assembly info file or do not specify /ensureassemblyinfo and create assembly info files manually");
                     }
                     continue;
                 }
 
                 if (name.IsSwitch("overrideconfig"))
                 {
-                    foreach (var item in value.Split(';'))
+                    var keyValueOptions = (value ?? "").Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
+                    if (keyValueOptions.Length == 0)
+                    {
+                        continue;
+                    }
+
+                    arguments.HasOverrideConfig = true;
+
+                    if (keyValueOptions.Length > 1)
                     {
-                        var configOverride = item.Split('=');
+                        throw new WarningException("Can't specify multiple /overrideconfig options: currently supported only 'tag-prefix' option");
+                    }
+
+                    // key=value
+                    foreach (var keyValueOption in keyValueOptions)
+                    {
+                        var keyAndValue = keyValueOption.Split(new[] { '=' }, StringSplitOptions.RemoveEmptyEntries);
+                        if (keyAndValue.Length != 2)
+                        {
+                            throw new WarningException(string.Format("Could not parse /overrideconfig option: {0}. Ensure it is in format 'key=value'", keyValueOption));
+                        }
 
-                        switch (configOverride[0])
+                        var optionKey = keyAndValue[0].ToLowerInvariant();
+                        switch (optionKey)
```

Comment:
The optimal way to match the `optionKey` with `Config` property, would be to go through the `Config` properties and find the one with a matching `YamlMemberAttribute.Alias`. That would ensure all config parameters can be overridden and is also more future-proof. If no matching property can be found, a `WarningException` should be thrown. It would be good to have tests for most of these overrides as well. Think you can look into that?